packages/zowe-explorer/src/uss/ZoweUSSNode.ts

```diff
@@ -484,7 +454,7 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
                     vscode.window.showInformationMessage(
                         localize(
                             "openUSS.name.exists",
-                            "There is already file with same name please change your OS file system settings and make case sensitive if you want edit this files"
+                            "There is already a file with the same name. Please change your OS file system settings if you want to give case sensitive file names"
```

Comment:
Same question as above - should we split across multiple lines to make the linter happy?

Reply:
See above comment regarding lint warnings